hexa/core/analytics.py

```diff
@@ -0,0 +1,87 @@
+from datetime import datetime
+
+from django.conf import settings
+from django.http import HttpRequest
+from mixpanel import BufferedConsumer, Mixpanel
+from sentry_sdk import capture_exception
+from ua_parser import user_agent_parser
+
+mixpanel_consumer = BufferedConsumer()
+
+
+def track(
+    user,
```

Comment:
Request is more important than user (you can get the user from the request if needed)